tests/console/yast2_dns_server_wizard_service_active_enabled.pm

```diff
@@ -0,0 +1,39 @@
+# SUSE's openQA tests
```

Comment:
let's not use `wizard` or those UI term, something functional would do:
`yast2_dns_server_initial_setup.pm`. wdyt?

Reply:
Ok, updated.